docs/cookbook/taxation/customize-tax-by-address.rst

```diff
@@ -0,0 +1,138 @@
+How to configure tax rates to be based on shipping address?
+===========================================================
+
+The default configuration of Sylius tax calculation is based on billing address but there are situations where we would
+like to use a shipping address to be used in this process. This could become necessary for anyone who uses Sylius in European Union
+as from 1st July 2021 the new taxation rules will be applied.
+
+.. note::
+
+    You can learn more about new EU taxation rules `here <https://ec.europa.eu/taxation_customs/business/vat/modernising-vat-cross-border-ecommerce_en>`_.
+
+To change the way how the taxes are calculated; by billing or by shipping address, you need to override the service called
+`OrderTaxesProcessor.php` from `Sylius/Component/Core/OrderProcessing/`. First let's copy code from original Processor to our service
+in ``src/OrderProcessing/OrderTaxesProcessor.php``
+
+.. code-block:: php
```

Comment:
I wonder if it wouldn't be better not to show the entire copied service because some things can change in future 🤔